INTERPRETER.s

```diff
@@ -404,7 +404,7 @@ ITR1		INDEX	LOC		# THE STORECODE WAS STORED COMPLEMENTED TO
 		AD	NEGONE		# (YUL CAN'T REMOVE 1 BECAUSE OF EARLY CCS)
```

Comment:
Non-spotted typo.

`CAN'T` should be `CANT`. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1119.jpg)